server/lib/report-manager.js

```diff
@@ -38,8 +40,13 @@ class ReportManager
             throw new KError("alert's account does not exist");
         }
 
+        let defaultCurrency = await Config.byName('defaultCurrency').value;
+
         let operationsByAccount = new Map;
         for (let a of accounts) {
+            // We get the currency for this account, to format amounts correctly
+            let currency = a.currency ? a.currency : defaultCurrency;
+            a.currencyFormatter = formatCurrency(currency);
```

Comment:
It looks like you don't actually need to store the `currencyFormatter` within the account here, as it's used only in one or two places (non critical to speed) below. Can you move this code down, please?